From 136d2309988f7c379f6439363b53c14404738d7a Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Thu, 14 Sep 2017 11:43:41 +0200
Subject: [PATCH 4/5] doc: firewall-cmd: Document --query-* options return
 codes

The "EXIT CODES" section didn't cover the fact that all --query-*
options return 1 if no error occurred but the query itself was not
successful.

Fixes: RHBZ#1372716
Signed-off-by: Phil Sutter <psutter@redhat.com>
---
 doc/xml/firewall-cmd.xml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/doc/xml/firewall-cmd.xml b/doc/xml/firewall-cmd.xml
index bdb5767634aaa..0b54b0be999c8 100644
--- a/doc/xml/firewall-cmd.xml
+++ b/doc/xml/firewall-cmd.xml
@@ -2281,6 +2281,9 @@ firewall-cmd --permanent --add-port=443/tcp
 	</tgroup>
       </informaltable>
     </para>
+    <para>
+      Note that return codes of <command>--query-*</command> options are special: Successful queries return 0, unsuccessful ones return 1 unless an error occurred in which case the table above applies.
+    </para>
   </refsect1>
 
   &seealso;
-- 
2.13.1

